service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerAboutToStartTest.java

```diff
@@ -91,6 +95,19 @@ void shouldNotAddAmountToPayFieldWhenFeatureToggleIsFalse() {
         assertThat(response.getData()).doesNotContainKey("amountToPay");
     }
 
+    @Test
+    void shouldNotDisplayAmountToPayFieldWhenErrorIsThrown() {
+        given(ldClient.boolVariation(eq("FNP"), any(), anyBoolean())).willReturn(true);
```

Comment:
as repeated couple times, can be extracted to method
givenPaymentEnabled(boolean enabled){
given(ldClient.boolVariation(eq("FNP"), any(), anyBoolean())).willReturn(enabled);
}